### PR TITLE
Whitesource SSO info

### PR DESCRIPTION
**Pull request by jwalker**
Whitesource info

### CODE REVIEW

File: _data/vendors.yml [hunk 1]
```diff
@@ -333,6 +333,15 @@
   updated_at: 2018-10-17
   footnotes: "[^victorops]: You can add SSO to a plan for $5 per u/m"
 
+- name: Whitesource
+  url: https://www.whitesourcesoftware.com/
+  base_pricing: $250 per month
+  sso_pricing: $3,0000 per year
```
Comment (NovaAtWarren) [inline _data/vendors.yml:339]: Might want to change this to $28,000, to reflect pricing changes and increase legibility.

File: _data/vendors.yml [hunk 2]
```diff
@@ -333,6 +333,15 @@
   updated_at: 2018-10-17
   footnotes: "[^victorops]: You can add SSO to a plan for $5 per u/m"
 
+- name: Whitesource
+  url: https://www.whitesourcesoftware.com/
+  base_pricing: $250 per month
```
Comment (NovaAtWarren) [inline _data/vendors.yml:338]: Base price was changed to $120 / year

**Review by NovaAtWarren — CHANGES_REQUESTED** [review]
Values out of date, otherwise good

**Comment by p-linnane** [comment]
WhiteSource has been replaced by Mend.io per https://www.whitesourcesoftware.com.

This PR can probably be closed.

**Pull request closed without merging**